integrated_channels/utils.py

```diff
@@ -185,7 +185,13 @@ def get_image_url(content_metadata_item):
     return image_url
 
 
-def is_already_transmitted(transmission, enterprise_enrollment_id, grade, subsection_id=None):
+def is_already_transmitted(
+    transmission,
+    enterprise_enrollment_id,
+    grade,
+    subsection_id=None,
+    detect_grade_updated=True,
```

Comment:
Genuinely curious, do we need this to have a default value?

Reply:
good q, I don't think so, it was a safety measure in case we didn't pass a value